dataRetriever/resolvers/genericBlockBodyResolver.go

```diff
@@ -52,7 +52,7 @@ func NewGenericBlockBodyResolver(
 
 // ProcessReceivedMessage will be the callback func from the p2p.Messenger and will be called each time a new message was received
 // (for the topic this validator was registered to, usually a request topic)
-func (gbbRes *genericBlockBodyResolver) ProcessReceivedMessage(message p2p.MessageP2P, _ func(buffToSend []byte)) error {
+func (gbbRes *genericBlockBodyResolver) ProcessReceivedMessage(message p2p.MessageP2P, fromConnectedPeer p2p.PeerID, _ func(buffToSend []byte)) error {
```

Comment:
fromConnectedPeer not used.

Reply:
Agree, some MesageProcessor implementations have the _ place holder, others have fromConnectedPeer. In the end (next PRs) all of them will need fromConnectedPeer as to be able to withstand the possible flooding.